pyensembl/sequence_data.py

```diff
@@ -20,33 +20,42 @@
 from Bio import SeqIO
 import datacache
 
-from .common import CACHE_SUBDIR, require_ensembl_id
-from .release_info import check_release_number
+from .common import (CACHE_SUBDIR, require_ensembl_id, is_url_format,
+                     copy_to_cache_if_needed, local_file_cache_path)
 
 
 class SequenceData(object):
-    """Container for reference nucleotide and amino acid sequenes.
+    """
+    Container for reference nucleotide and amino acid sequenes.
 
     Downloads and caches FASTA file, creates database mapping
     identifiers to sequences.
     """
     def __init__(
             self,
-            ensembl_release,
-            fasta_url,
+            genome_source,
```

Comment:
I went back and forth on this. On the one hand, `GenomeSource` specifies the install strings that I use in error messages here, so the class itself is handy. (And whether those install strings, e.g. `pyensembl install --gtf-path` should specify all the GTF/FASTA information, is unclear in my mind.) On the other hand, I agree that the rest of the data isn't really needed. This seemed fine enough, even if not perfect?